src/__tests__/html-text-input-test.js

```diff
@@ -100,7 +100,7 @@ describe( 'HTMLInputView', () => {
 
 		const wrapper = shallow(
 			<HTMLInputView
-				setTitleAction={ setTitleAction }
+				editTitle={ setTitleAction }
```

Comment:
Will this prop set override the `editTitle` provided inside the `HTMLInputView` itself via `withDispatch`?

Reply:
Yes because we're not importing the default `HTMLInputView` component [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/e7629454d96300d772af5c0a1488c3ddf654fa4b/src/__tests__/html-text-input-test.js#L9) but the original class exported [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/e7629454d96300d772af5c0a1488c3ddf654fa4b/src/components/html-text-input.js#L40)

Reply:
Ah, I see. Gosh, that pattern is super confusing :(. I wish we could make it better.